content/client-lib-development-guide/features.textile

```diff
@@ -97,7 +97,11 @@ h3(#restclient). RestClient
 * @(RSC12)@ REST endpoint host is configurable in the Client constructor with the option @restHost@
 * @(RSC13)@ The client library must use the connection and request timeouts specified in the @ClientOptions@, falling back to the defaults described in @ClientOptions@ below
 * @(RSC15)@ Host Fallback
-** @(RSC15b)@ The fallback behavior described below only applies when either @ClientOptions#restHost@ has not been set to an explicit value (see "RSC11a":#RSC11a), the deprecated @ClientOptions#fallbackHostsUseDefault@ option is set to @true@, or an array of @ClientOptions#fallbackHosts@ is provided. If host fallback does not apply, failing HTTP requests that would have "qualified for a retry against a fallback host (see RSC15d)":#RSC15d, will instead result in an error immediately
+** @(RSC15b)@ The fallback behavior described below only applies when either:
+*** @(RSC15b1)@ @ClientOptions#restHost@ has not been set to an explicit value (see "RSC11a":#RSC11a) AND @ClientOptions#port@ is not set AND @ClientOptions#tlsPort@ is not set
```

Comment:
I don't think the capitalisation of instances of "and" is necessary. From what I can see we don't have a convention, informal or otherwise, of doing that elsewhere in this document.

```suggestion
*** @(RSC15b1)@ @ClientOptions#restHost@ has not been set to an explicit value (see "RSC11a":#RSC11a) and @ClientOptions#port@ is not set and @ClientOptions#tlsPort@ is not set
```

Reply:
Updated in d3ac5350